core/dyn-configs/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+#![doc = include_str!("../README.md")]
+
+use lazy_static::*;
+use std::sync::atomic::{AtomicU64, Ordering};
+
+lazy_static! {
```

Comment:
A while ago the codebase migrated from using `lazy_static` to `once_cell::sync::Lazy`. Please update it to be using `once_cell::sync::Lazy`